library/src/main/kotlin/it/krzeminski/githubactions/dsl/WorkflowBuilder.kt

```diff
@@ -107,7 +139,7 @@ fun workflow(
 
 private fun List<Job>.requireUniqueJobNames() {
     val countPerJobName = this
```

Comment:
```suggestion
    val countPerJobId = this
```

and similar other places below, I cannot add a comment on them (GitHub's limitation?).